src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CcdApi.java

```diff
@@ -320,16 +324,32 @@ public StartEventResponse startEventForAttachScannedDocs(
                 caseRef,
                 eventTypeId
             );
+
+            log.info("Started event in CCD. Event: {}, case type: {}. {}", eventTypeId, caseTypeId, logContext);
+
+            CaseDataContent caseData = caseDataContentBuilder.apply(eventResponse);
+
+            feignCcdApi.submitEventForCaseWorker(
+                authenticator.getUserToken(),
+                authenticator.getServiceToken(),
+                authenticator.getUserDetails().getId(),
+                jurisdiction,
+                caseTypeId,
+                caseRef,
+                true,
+                caseData
+            );
         } catch (FeignException.NotFound e) {
             throw new UnableToAttachDocumentsException(
                 String.format(
-                    "Attach documents start event failed for case type: %s and case ref: %s", caseTypeId, caseRef
+                    "Event failed. Event: %s, case type: %s, case ref: %s", eventTypeId, caseTypeId, caseRef
                 ),
                 e
             );
         } catch (FeignException e) {
-            debugCcdException(log, e, "Failed to call 'startEventForCaseWorker'");
+            debugCcdException(log, e, "Failed to call 'eventForCaseWorker'");
```

Comment:
'eventForCaseWorker' can be replaced with `eventTypeId` variable?

Reply:
aye, that was my first version but noticed there's extra catch specific to event. so renamed method but not debug message. 1 sec